packages/yourloops/test/integration/pages/trends/trends-view-for-caregiver.spec.tsx

```diff
@@ -58,9 +58,8 @@ describe('Trends view for caregiver', () => {
   it('should render correct layout', async () => {
     mockDataAPI(minimalTrendViewData)
     renderPage(`/patient/${unmonitoredPatientId}/trends`)
-
-    expect(await screen.findByTestId('patient-data-subnav-outer', {}, { timeout: 3000 })).toBeVisible()
-    checkPatientNavBarAsCaregiver(false)
+    expect(await screen.findByTestId('patient-nav-bar', {}, { timeout: 3000 })).toBeVisible()
```

Comment:
useless timeout

Reply:
Are we sure? This always works locally but this was done for CI purpose. Maybe now that we are on github action, it might work...